layouts/partials/docs-nav.html

```diff
@@ -17,7 +17,31 @@
         {{ end -}}
         <span>{{.LinkTitle}}</span>
       </a>
+      {{if (eq (.Params.linkTitle) "Redis for Kubernetes")}}
+      <div id="versionSelectorKubernetes" class="menu__version-selector version-selector-control" onclick="_openVersionSelector('Kubernetes')" style="display: none;">
+        <button class="menu__version-selector-btn version-selector-control">
+            <span id="versionSelectorKubernetesValue" class="version-selector-control">latest</span>
+            <span class="menu__version-selector__toggler opener version-selector-control">&#x25BC;</span>
+            <span class="menu__version-selector__toggler closer version-selector-control">&#x25B2;</span>
+        </button>
+        <div id="versionDropdownKubernetes" class="menu__version-selector__list version-selector-control">
+            <a href="https://dev.learn.redis.com/docs/latest/operate/kubernetes/" id="kubernetes-version-select-latest" onclick="_setSelectedVersion('kubernetes', 'latest')">latest</a>
```

Comment:
Do the links need to be changed from `dev.learn.redis.com` to `redis.io`? Or is this just for testing/staging purposes?
```suggestion
            <a href="https://redis.io/docs/latest/operate/kubernetes/" id="kubernetes-version-select-latest" onclick="_setSelectedVersion('kubernetes', 'latest')">latest</a>
```

Reply:
Well spotted, thank you